src/scriptdev2/scripts/outland/hellfire_citadel/hellfire_ramparts/boss_watchkeeper_gargolmar.cpp

```diff
@@ -16,8 +16,8 @@
 
 /* ScriptData
 SDName: Boss_Watchkeeper_Gargolmar
-SD%Complete: 80
-SDComment: Missing adds to heal him. Surge should be used on pTarget furthest away, not random.
+SD%Complete: 90
+SDComment: Missing adds to heal him. Surge spell implementation needs some improvements (No charge animation)
```

Comment:
This charge animation is bigger problem, see here https://github.com/cmangos/issues/issues/697